src/index.html

```diff
@@ -1,13 +1,47 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+
+      <img class="card__image"
+        src="images/imac.jpeg"
+        alt="">
```

Comment:
```suggestion
      <img 
        class="card__image"
        src="images/imac.jpeg"
        alt="описание картинки"
      >
```
Форматируй теги таким образом, и не оставляй alt пустым